Apps/Database/src/Delegates/IEmailDelegate.cs

```diff
@@ -0,0 +1,75 @@
+﻿// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.Database.Delegates
+{
+    using System;
+    using System.Collections.Generic;
+    using HealthGateway.Database.Models;
+
+    /// <summary>
+    /// Interface for sending email.
+    /// </summary>
+    public interface IEmailDelegate
+    {
+        /// <summary>
+        /// Fetches the email object from the database.
+        /// </summary>
+        /// <param name="emailId">The Email ID to retrieve.</param>
+        /// <returns>The found Email object.</returns>
+        Email GetEmail(Guid emailId);
+
+        /// <summary>
+        /// Gets an email in the new state with standard priority or higher.
+        /// </summary>
+        /// <param name="emailId">The Email ID to retrieve.</param>
+        /// <returns>The found Email object.</returns>
+        Email GetNewEmail(Guid emailId);
+
+        /// <summary>
+        /// Gets a list of New low priority emails to process.
```

Comment:
The implementation also sorts the emails. Having that in the documentation of the interface would be helpful